src/tools/install-git-hook/src/pre-commit.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# Call `tidy --bless` before each commit
+#
+# To enable this hook, run `./x.py run install-git-hook`.
+# To disable it, run `./x.py run install-git-hook --remove`
+set -Eeuo pipefail
```

Comment:
This will break if the user is running `bash` and not `sh`. You should either remove `-o pipefail` or change the shebang to `/usr/bin/env bash`. `shellcheck` can pick up on this sort of thing automatically, I highly recommend it.

Reply:
Woah that's a super cool tool. Thanks!